.github/workflows/pr.yml

```diff
@@ -0,0 +1,101 @@
+name: Go Test on Pull Requests
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+jobs:
+  go:
+    name: Check sources
+    runs-on: ubuntu-20.04
+    container: fedora:35
+    env:
+      OPERATOR_SDK_VERSION: v1.18.0
+    steps:
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Cache Operator SDK ${{ env.OPERATOR_SDK_VERSION }}
```

Comment:
Silly comment, but could you move this one step down so all the operator sdk stuff is together?